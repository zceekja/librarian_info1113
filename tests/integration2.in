add collection book1.csv
add member Tony Stark
add member Peter Parker
rent 100000 111111
rent 100000 111112
rent 100000 111113
rent 100000 111114
rent 100001 111111
relinquish 100000 111111
rent 100001 111111
relinquish all 100001
common 100000 100001
exit