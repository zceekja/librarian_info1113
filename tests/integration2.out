user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Book is currently unavailable.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Ending Library process.
